openvdb_houdini/CMakeLists.txt

```diff
@@ -334,6 +364,28 @@ if(USE_BLOSC OR OpenVDB_USES_BLOSC)
   endif()
 endif()
 
+##########
+# OPcustomize
+##########
+
+if(OPENVDB_HOUDINI_OPCUSTOMIZE)
+  # the run-time environment variable OPENVDB_OPHIDE_POLICY takes precedence over
+  # the compile-time flag, so define this as the policy is not implemented at
+  # compile-time if OPcustomize is in use
+  if (OPENVDB_HOUDINI_DEFAULT_POLICY)
+    set(OPCUSTOMIZE_ENV "OPENVDB_OPHIDE_POLICY=${OPENVDB_HOUDINI_DEFAULT_POLICY}")
+  else()
+    set(OPCUSTOMIZE_ENV "")
+  endif()
+  add_custom_target(opcustomize ALL
+    COMMAND ${CMAKE_COMMAND} -E env ${OPCUSTOMIZE_ENV}
+      hython ${CMAKE_CURRENT_SOURCE_DIR}/vdbopcustomize.py ${SOP_TARGET_FILES}
+      > ${CMAKE_CURRENT_BINARY_DIR}/OPcustomize
```

Comment:
HScript is deprecated, so it would be preferable to generate a Python script instead.  Installing it as `${OPENVDB_HOUDINI_INSTALL_PREFIX}/python2.7libs/pythonrc.py` should cause it to be executed at startup.